lib/exception_handling.rb

```diff
@@ -214,21 +203,41 @@ def write_exception_to_log(ex, exception_context, timestamp)
       end
     end
 
+    #
+    # Send notifications to configured external services
+    #
+    def send_external_notifications(exception_info)
+      results = {}
+      if honeybadger?
+        results[:honeybadger_status] = send_exception_to_honeybadger(exception_info)
```

Comment:
should this `if` set the honeybadger_status to skipped if `!honeybadger?`

Reply:
Later on, if `:honeybadger_status` is nil, I default it to `:skipped`. 

I'm not sure if we actually care about `:skipped` versus `nil`. If we do, then I can make sure `:skipped` is only set if we actually intended on skipping. Currently, I think they basically represent the same thing.

I make a point in `send_exception_to_honeybadger` to return `:skipped` explicitly so that future devs know that the return value matters for that method.

Reply:
👍 sounds good, thank you for the back and forth on this.